From d8de6f0778e5ae8162651754b3bd881e767b77a7 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Sat, 28 Apr 2012 16:32:53 -0300
Subject: [PATCH 07/40] 4.1.3 Quel'Delar: Isla de Quel'Danas
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- Añadido phasing durante las últimas misiones de la cadena
- Añadidos algunos NPC faltantes en dicha fase
---
 .../reanemu/2012_03_26_103_world_quel_danas.sql    |  104 ++++++++++++++++++++
 1 files changed, 104 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_03_26_103_world_quel_danas.sql

diff --git a/sql/updates/world/reanemu/2012_03_26_103_world_quel_danas.sql b/sql/updates/world/reanemu/2012_03_26_103_world_quel_danas.sql
new file mode 100644
index 0000000..a5fce2f
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_03_26_103_world_quel_danas.sql
@@ -0,0 +1,104 @@
+/********************************
+         Missing Spawns
+********************************/
+
+-- Creatures
+DELETE FROM `creature` WHERE `id` IN (37527,37523,37552,37763,37764,37781);
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`) VALUES
+(NULL, 37527, 530, 1, 2048, 0, 0, 12559.2, -6783.39, 14.9885, 6.2551, 300, 0, 0, 558900, 25240, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12556.6, -6769.78, 15.1021, 6.25903, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12556.3, -6779.49, 15.0456, 6.2661, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12567.7, -6770.48, 15.0915, 4.7228, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12567.8, -6779.31, 15.0915, 1.53016, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12576.3, -6779.83, 15.0915, 1.55136, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12576.4, -6770.46, 15.0915, 4.69296, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12590.6, -6783.36, 14.8457, 6.25983, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37523, 530, 1, 2048, 0, 0, 12590.9, -6767.48, 14.911, 6.27947, 300, 0, 0, 41, 60, 0, 0, 0, 0),
+(NULL, 37552, 530, 1, 2048, 0, 0, 11794.4, -7063.62, 25.4531, 3.2549, 300, 0, 0, 123, 180, 0, 0, 0, 0),
+(NULL, 37763, 580, 1, 2048, 0, 0, 1675.76, 600.142, 28.0502, 5.65094, 300, 0, 0, 8200, 1200, 0, 0, 0, 0),
+(NULL, 37764, 580, 1, 2048, 0, 0, 1672.19, 595.356, 28.077, 5.79624, 300, 0, 0, 16400, 1800, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1759.89, 673.405, 71.1906, 2.24314, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1764.6, 676.808, 71.1906, 2.29027, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1773.87, 684.437, 71.1906, 2.23922, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1777.24, 686.642, 71.1906, 2.21251, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1767.55, 697.223, 71.1906, 3.90504, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1747.7, 685.158, 71.1906, 0.845907, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1743.79, 687.692, 71.1906, 0.991203, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1748.9, 709.424, 71.1906, 4.20505, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1726.28, 697.03, 71.1906, 1.20797, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1720.16, 698.983, 71.1906, 1.35405, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1726.98, 716.699, 71.1906, 4.43438, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1704.35, 719.792, 71.0384, 4.69748, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1688.73, 606.871, 28.3971, 4.41473, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1701.54, 605.022, 28.4091, 4.85847, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1678.66, 605.222, 28.0503, 6.03657, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1673.96, 602.873, 28.0503, 5.95804, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1670.24, 597.614, 28.2006, 5.70199, 300, 0, 0, 126, 0, 0, 0, 0, 0),
+(NULL, 37781, 580, 1, 2048, 0, 0, 1669.52, 592.665, 29.0599, 5.74911, 300, 0, 0, 126, 0, 0, 0, 0, 0);
+-- Gameobject
+DELETE FROM `gameobject` WHERE `id`=201794;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES
+(NULL, 201794, 580, 1, 2048, 1695.57, 609.515, 29.2366, 1.40668, 0, 0, 0.646769, 0.762686, -120, 0, 1);
+
+/******************************
+    Quel'delar phasing
+******************************/
+
+SET @PHASE := 70193;
+
+DELETE FROM `spell_area` WHERE `spell`=@PHASE;
+INSERT INTO `spell_area` (`spell`,`area`,`quest_start`,`quest_start_active`,`quest_end`,`aura_spell`,`racemask`,`gender`,`autocast`) VALUES
+-- Alliance
+(@PHASE,4080,24522,1,24553,0,1101,2,1),
+(@PHASE,4081,24522,1,24553,0,1101,2,1), 
+(@PHASE,4082,24522,1,24553,0,1101,2,1),
+(@PHASE,4083,24522,1,24553,0,1101,2,1),
+(@PHASE,4084,24522,1,24553,0,1101,2,1),
+(@PHASE,4085,24522,1,24553,0,1101,2,1),
+(@PHASE,4086,24522,1,24553,0,1101,2,1),
+(@PHASE,4087,24522,1,24553,0,1101,2,1),
+(@PHASE,4088,24522,1,24553,0,1101,2,1),
+(@PHASE,4089,24522,1,24553,0,1101,2,1),
+(@PHASE,4090,24522,1,24553,0,1101,2,1),
+(@PHASE,4091,24522,1,24553,0,1101,2,1),
+(@PHASE,4092,24522,1,24553,0,1101,2,1),
+(@PHASE,4093,24522,1,24553,0,1101,2,1),
+(@PHASE,4094,24522,1,24553,0,1101,2,1),
+(@PHASE,4095,24522,1,24553,0,1101,2,1),
+(@PHASE,4075,24522,1,24553,0,1101,2,1),
+-- Horde, non bloodelves
+(@PHASE,4080,24562,1,24564,0,178,2,1),
+(@PHASE,4081,24562,1,24564,0,178,2,1),
+(@PHASE,4082,24562,1,24564,0,178,2,1),
+(@PHASE,4083,24562,1,24564,0,178,2,1),
+(@PHASE,4084,24562,1,24564,0,178,2,1),
+(@PHASE,4085,24562,1,24564,0,178,2,1),
+(@PHASE,4086,24562,1,24564,0,178,2,1),
+(@PHASE,4087,24562,1,24564,0,178,2,1),
+(@PHASE,4088,24562,1,24564,0,178,2,1),
+(@PHASE,4089,24562,1,24564,0,178,2,1),
+(@PHASE,4090,24562,1,24564,0,178,2,1),
+(@PHASE,4091,24562,1,24564,0,178,2,1),
+(@PHASE,4092,24562,1,24564,0,178,2,1),
+(@PHASE,4093,24562,1,24564,0,178,2,1),
+(@PHASE,4094,24562,1,24564,0,178,2,1),
+(@PHASE,4095,24562,1,24564,0,178,2,1),
+(@PHASE,4075,24562,1,24564,0,178,2,1),
+-- Blood elf
+(@PHASE,4080,24562,1,24594,0,512,2,1),
+(@PHASE,4081,24562,1,24594,0,512,2,1),
+(@PHASE,4082,24562,1,24594,0,512,2,1),
+(@PHASE,4083,24562,1,24594,0,512,2,1),
+(@PHASE,4084,24562,1,24594,0,512,2,1),
+(@PHASE,4085,24562,1,24594,0,512,2,1),
+(@PHASE,4086,24562,1,24594,0,512,2,1),
+(@PHASE,4087,24562,1,24594,0,512,2,1),
+(@PHASE,4088,24562,1,24594,0,512,2,1),
+(@PHASE,4089,24562,1,24594,0,512,2,1),
+(@PHASE,4090,24562,1,24594,0,512,2,1),
+(@PHASE,4091,24562,1,24594,0,512,2,1),
+(@PHASE,4092,24562,1,24594,0,512,2,1),
+(@PHASE,4093,24562,1,24594,0,512,2,1),
+(@PHASE,4094,24562,1,24594,0,512,2,1),
+(@PHASE,4095,24562,1,24594,0,512,2,1),
+(@PHASE,4075,24562,1,24594,0,512,2,1);
-- 
1.7.8.msysgit.0

